data/changelog.txt

```diff
@@ -1,3 +1,104 @@
+# 0.F (05-01-2020)
+
+## Features:
+Added option to randomize INITIAL_DAY.
+Vehicle autopilot follow.
+Monfaction attitude hate.
+Allows mutations to cast spells via enchantments.
+Mining multi-activity for NPCs and players.
+Basecamp job rework - wandering/sitting/job priorities.
+Auto eat and drink from auto-zones during long activities/waiting.
+Flyable Helicopters.
+Towing vehicles.
+User defined height and age.
+Add support for json-defined achievements to provide more goals to strive for.
+Lock picking overhaul.
+Rot away items away in reality bubble.
+Implementing use of system time for holidays.
+
+## Content:
+Gun volume and weight audit.
+Move most urban development houses to vanilla.
+Add Survivor RV and car.
+Populate rooftops with monsters.
+Large addition of chemicals, chemistry tools and recipes.
+
+## Interface:
+Introduce and migrate most windows to ui_adaptor.
+Crafting GUI - Highlight hidden recipe info
+
+## Mods:
+[Magiclysm] Manatouched mutation tree
+Dark Skies: Reworks Strays
+Dark Skies: Remasters the Order
```

Comment:
Dark Skies Above didn't exist in 0.E, so these don't mean much to them.
Perhaps:
```suggestion
Add Dark Skies Above, a partial conversion mod based around an alien invasion rather than a zombie apocalypse.
```